app/models/finance/statement.rb

```diff
@@ -6,7 +6,7 @@ class Finance::Statement < ApplicationRecord
   self.table_name = "statements"
 
   belongs_to :cpd_lead_provider
-
+  belongs_to :cohort, optional: true
```

Comment:
is there no association between the statement and the contract? given a statement how would you fetch the contract?